pkg/client/local/local_test.go

```diff
@@ -0,0 +1,117 @@
+package local
```

Comment:
If you are not testing internal methods, please use `local_test` and import the `local` package here. I haven't tell you in other PRs, sorry.